test/documentation/seqan3-doxygen-package.cmake

```diff
@@ -0,0 +1,21 @@
+# -----------------------------------------------------------------------------------------------------
+# Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+# Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+# This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+# shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+# -----------------------------------------------------------------------------------------------------
+
+cmake_minimum_required (VERSION 3.7)
+
+set (CPACK_GENERATOR "TXZ")
+
+set (CPACK_COMPONENTS_ALL doc)
+
+set (CPACK_ARCHIVE_COMPONENT_INSTALL ON)
+
+set (CPACK_PACKAGE_VENDOR "seqan")
+# A description of the project, used in places such as the introduction screen of CPack-generated Windows installers.
+# set (CPACK_PACKAGE_DESCRIPTION_FILE "") # TODO
```

Comment:
There is a open TODO?

Reply:
yep, it is also an open todo in

https://github.com/seqan/seqan3/blob/a1d5d4d98ab82fddeed6075f1f645a956233822b/build_system/seqan3-package.cmake#L16

We don't have any windows installer yet, so I don't know what an appropriate value would be. I can remove the line if you want :)

Reply:
No, when its still open, and has nothing todo with this PR, than leave it open.